includes/rest-api/Controllers/Version3/class-wc-rest-order-refunds-controller.php

```diff
@@ -110,6 +110,13 @@ public function get_item_schema() {
 			'readonly'    => true,
 		);
 
+		$schema['properties']['api_restock'] = array(
+			'description' => __( 'When true, items are restocked.', 'woocommerce' ),
```

Comment:
Minor suggestion, can we update this description so that it more closely matches the wording shown within the refund interface?

```suggestion
			'description' => __( 'When true, refunded items are restocked.', 'woocommerce' ),
```